src/gui/BoardComponent.cpp

```diff
@@ -158,17 +168,22 @@ void BoardComponent::processorAdded (BaseProcessor* newProc)
     repaint();
 }
 
-void BoardComponent::processorRemoved (const BaseProcessor* proc)
+void BoardComponent::processorPrepareToRemove (const BaseProcessor* proc)
 {
     for (int i = cables.size() - 1; i >= 0; --i)
     {
         if (cables[i]->startProc == proc || cables[i]->endProc == proc)
             cables.remove (i);
     }
 
-    auto* editor = findEditorForProcessor (proc);
```

Comment:
Debating going back to old way of only having a `processorRemoved()` listener...